### PR TITLE
Show version name while deleting

### LINKED ISSUE #4575: While deleting a version, show the name of version in the confirmation modal.
### Is your proposal related to a problem?

Currently, when you delete a version from the app. The modal that shows doesn't confirm which version is being deleted.

### Describe the solution you'd like

Display the version that the user is trying to delete in the modal.

### Steps to reproduce the issue
- Create/Open an app in ToolJet
- Create a version and try to delete it
- Check the warning in the delete modal that pops-up

Screenshot 2022-10-23 at 6 57 49 PM

### PR DESCRIPTION
**Pull request by FroostyCodes**
Fixes #4575

### CODE REVIEW

**Comment by gsmithun4** [comment]
deployed https://tj-4589.herokuapp.com/

**Comment by FroostyCodes** [comment]
Screenshot for reference:
Screen Shot 2022-10-24 at 8 43 58 PM

File: frontend/src/Editor/AppVersionsManager.jsx [hunk 1]
```diff
@@ -283,7 +283,11 @@ export const AppVersionsManager = function AppVersionsManager({
               </div>
               <Confirm
                 show={showVersionDeletionConfirmation}
-                message={t('editor.appVersionManager.deleteVersion', 'Do you really want to delete this version?')}
+                message={t(
+                  'editor.appVersionManager.deleteVersion',
+                  'Do you really want to delete this version ({{version}})?',
+                  { version: appVersions.find((version) => version.id === deletingVersionId)?.name ?? '' }
+                )}
```
Comment (kavinvenkatachalam) [inline frontend/src/Editor/AppVersionsManager.jsx:290]:
```suggestion
                  { version: deletingVersion?.name ?? '' }
```

File: frontend/src/Editor/AppVersionsManager.jsx [hunk 2]
```diff
@@ -283,7 +283,11 @@ export const AppVersionsManager = function AppVersionsManager({
...
                 confirmButtonLoading={isDeletingVersion}
                 onConfirm={(versionId) => deleteAppVersion(versionId)}
                 queryConfirmationData={deletingVersionId}
```
Comment (kavinvenkatachalam) [inline frontend/src/Editor/AppVersionsManager.jsx:293]:
```suggestion
                queryConfirmationData={deletingVersion.id}
```

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@FroostyCodes Thanks for the PR.

Instead of looping and finding the name of the version, we can maintain it in the state. We already have a `deletingVersionId` state which can be changed to `deletingVersion` and maintain both `{ id: version.id, name: version.name }` in the state. This avoids an unnecessary loop to find the name of the version.

**Review by kavinvenkatachalam — APPROVED** [review]
LGTM

**Comment by withshubh** [comment]
Hey @FroostyCodes 
Thanks for contributing to ToolJet. :raised_hands: 
Please submit the form to claim your Hacktoberfest swag: https://airtable.com/shr1qR6xhqsWO7LD7

**Pull request merged**